src/index.ts

```diff
@@ -1 +1,3 @@
 export { getExtensions, watchExtensions, initializeKiltExtensionAPI } from './getExtension'
+export * from './messaging'
+export * from './quote'
```

Comment:
I think we should namespace these two
```suggestion
export * as Message from './messaging'
export * as Quote from './quote'
```